tools/mdanalysis/angle.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="mdanalysis_angle" name="Angle Analysis" version="@VERSION@">
+    <description>Time series of Angles</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>   
+    <expand macro="requirements" />
+    <command detect_errors="exit_code">
+<![CDATA[
+    python '$__tool_directory__/md_analysis_angle.py' 
+        --idcd '$dcdin'
```

Comment:
@tsenapathi most of these parameters are redundant in every tools maybe move them to macros?